source-browser.html

```diff
@@ -62,6 +87,44 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
     <a class="fixed right-0 bottom-0 mr2 mb2 text-decoration-none h2" id="refresh" href="#">🔄</a>
 
     <script type="text/javascript" src="https://unpkg.com/zepto@1.2.0/dist/zepto.js" integrity="sha384-Cp3V2nlfJJ5aA0ctd1PkfNAEMkXM0EGa6RrmEgiv8D6TCaeZJfUFs/PYfR+B5F5H" crossorigin="anonymous"></script>
+    <script type="text/javascript">
+      setInterval(function(){ 
```

Comment:
Please run this code through Prettier. There are many issues.

Reply:
Done!